src/app/components/GuestTickets.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+import { useState } from 'react';
+import FormInput from '@/app/components/FormInput';
+import Link from 'next/link';
+
+export default function signup(props: any) {
+  const [guestData, setGuestData] = useState<{
+    name: string;
+    email: string;
+    number: string;
+    tickets: string;
+  }>({
+    name: '',
+    email: '',
+    number: '',
+    tickets: `${props.tickets}`,
+  });
+  const inputs = [
+    {
+      id: 1,
```

Comment:
In the design it looks like this but we do not have anywhere in the database to store name and phone for guest users as it is right now. We are just storing the email and a think that that could be enough. We could skip the phone number, send the email with the ticket booking and use the name as a greeting in the end. For example: "Thank you, Markus". And not save the name in the database. What do you think about that?

Reply:
As before, we can use React for that.